Tests/VisualRecognitionV3Tests/VisualRecognitionTests.swift

```diff
@@ -173,8 +175,16 @@ class VisualRecognitionTests: XCTestCase {
 
         var classifierID: String?
         let expectation1 = expectation(description: "Train a classifier with only positive examples.")
-        visualRecognition.createClassifier(name: name, positiveExamples: classes, failure: failWithError) {
-            classifier in
+        visualRecognition.createClassifier(name: name, positiveExamples: classes) {
+            response, error in
+            if let error = error {
+                XCTFail("Unexpected error response from service: \(error)")
+                return
+            }
+            guard let classifier = response?.result else {
+                XCTFail("Unexpected response value")
```

Comment:
I think we should reword this to `Missing response value`.

Reply:
Agreed.  I'll make that change.